CLAUDE.md

```
@@ -0,0 +1,137 @@
+```bash
+# 复制示例环境变量并配置
+copy .env.example .env
+```
+
+必需环境变量：
+- `ANTHROPIC_API_KEY` 或 `ARK_API_KEY`：至少配置一个 LLM 凭证
+- `EMBEDDING_SERVICE`：`openai`（默认）或 `local`
+
+## 架构：DDD 四层
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)